docs/api.md

```diff
@@ -151,7 +151,7 @@ Puppeteer looks for certain environment variables to aid its operations:
 
 - `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` - defines HTTP proxy settings that are used to download and run Chromium.
 - `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` - do not download bundled Chromium during installation step.
-
+> Don't know how to set Environment Variable? Read [this](https://en.wikipedia.org/wiki/Environment_variable#Assignment)
```

Comment:
I'd rather linkify the "environment variables" on line 150

Reply:
Yeah that was better, thanks. I already made a change